gdal/frmts/gtiff/cogdriver.cpp

```diff
@@ -1011,6 +1011,36 @@ GDALDataset* GDALCOGCreator::Create(const char * pszFilename,
          }
     }
 
+    const char* osOvrCompress = CSLFetchNameValueDef(papszOptions, "OVERVIEW_COMPRESS", "AUTO");
+    if ( !EQUAL(osOvrCompress, "AUTO") ) 
+    {
+        CPLSetConfigOption("COMPRESS_OVERVIEW", osOvrCompress);
+    }
+
+    const char* osOvrQuality = CSLFetchNameValueDef(papszOptions, "OVERVIEW_QUALITY", "AUTO");
+    if ( !EQUAL(osOvrQuality, "AUTO") ) 
+    {
+        CPLSetConfigOption("JPEG_QUALITY_OVERVIEW", osOvrCompress);
+        CPLSetConfigOption("WEBP_LEVEL_OVERVIEW", osOvrCompress);
+    }
+
+    const char* osOvrPredictor = CSLFetchNameValueDef(papszOptions, "OVERVIEW_PREDICTOR", "FALSE");
```

Comment:
This was copied from the other `PREDICTOR` setting parsing just above this piece of code